payments/models.py

```diff
@@ -116,29 +115,34 @@ def _save_new(self, *args, **kwargs):
         else:
             raise Exception('Cannot obtain a new product ID for product {}.'.format(self))
 
-    def get_pretax_price_for_reservation(self, reservation: Reservation, rounded: bool = True,
-                                         as_sub_units: bool = False) -> Price:
+    def get_pretax_price_for_time_range(self, begin: datetime, end: datetime, rounded: bool = True) -> Decimal:
+        assert begin <= end
```

Comment:
Does it make sense to allow equality here?

Reply:
My initial point was to disallow times that cause incorrect results, and when the times are the same, it still "works" by giving result 0. Then again, you are correct that using it with those equal values does not really make sense, so probably just better to disallow that.

Reply:
Fixed.